torch_geometric/explain/algorithm/captum_explainer.py

```diff
@@ -70,6 +75,20 @@ def forward(
         **kwargs,
     ) -> Explanation:
 
+        # Check if the method supports multiple indices.
+        if isinstance(index, list) and len(index) > 1:
```

Comment:
Following the `Explainer` inteface, `index` needs to be `int` or `Tensor`, so we should check via `index.numel() > 1`